src/components/JoinUs/Perks.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import styled from 'styled-components'
+import remcalc from 'remcalc'
+import breakpoint from 'styled-components-breakpoint'
+import { Row, Col } from '../grid'
+import { Padding } from 'styled-components-spacing'
+import { Paragraph, H2 } from '../Typography'
+import Image from '../Common/Image'
+
+import { Section, ShiftedColumn } from './elements'
+
+const PerksListContainer = styled(ShiftedColumn)`
+  display: grid;
+  row-gap: ${remcalc(24)};
+  grid-template-columns: 1fr 1fr;
+  column-gap: 42px;
+  padding-top: ${remcalc(24)};
+
+  ${breakpoint('smallPhone')`
+    grid-template-columns: 1fr 1fr;
+  `};
+
+  ${breakpoint('smallTablet')`
+    grid-template-columns: 1fr 1fr 1fr;
+  `};
+
+  ${breakpoint('tablet')`
+    grid-template-columns: 1fr 1fr;
+    row-gap: ${remcalc(36)};
+    column-gap: 42px;
+    padding-top: ${remcalc(12)};
+  `};
+`
+
+const Icon = styled(Image)`
+  width: 60px;
+  height: 60px;
+`
+
+const PerkContainer = styled(Padding)`
+  display: flex;
+  flex-direction: column;
+`
+
+const PerkDescription = styled(Paragraph)`
+  padding: ${remcalc(12)} 0;
+`
+const Perk = ({ icon, description }) => (
+  <PerkContainer>
+    <Icon image={icon} />
+    <PerkDescription noMargin>{description}</PerkDescription>
+  </PerkContainer>
+)
+
+const PerksList = ({ perks }) => (
```

Comment:
I think we should avoid naming things `List` unless they are semantically lists under the hood?

Arguably, perhaps these could/should be a list?

Reply:
You're right, they should be a list

Reply:
done! 👍